.idea/runConfigurations/Tags__in_process__.xml

```diff
@@ -0,0 +1,22 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
Is this checked in on purpose? Should .idea be ignored

Reply:
File is to support pre loaded config for E2E tests

Reply:
It was intentional to make the runConfigurations available to other developers